Changelog.md

```diff
@@ -2,6 +2,49 @@
 
 ### libsdformat 10.X.X (202X-XX-XX)
 
+### libsdformat 10.2.0 (2021-01-12)
+
+1. Disable ign test on Windows
+    + [Pull request 456](https://github.com/osrf/sdformat/pull/456)
+
+1. Add Heightmap class
+    + [Pull request 388](https://github.com/osrf/sdformat/pull/388)
+
+1. Added render\_order to materisl
+    + [Pull request 446](https://github.com/osrf/sdformat/pull/446)
+
+1. Store material file path information.
+    + [Pull request 349](https://github.com/osrf/sdformat/pull/349)
+
+1. Support nested models in DOM and frame semantics.
+    * [Pull request 316](https://github.com/osrf/sdformat/pull/316)
+    + [Pull request 341](https://github.com/osrf/sdformat/pull/341)
+
+1. Find python3 in cmake, fix cmake warning.
+    * [Pull request 328](https://github.com/osrf/sdformat/pull/328)
+
+1. Fix Actor copy operators and increase test coverage.
+    * [Pull request 301](https://github.com/osrf/sdformat/pull/301)
+
+1. GitHub Actions CI, pull request labels.
+    * [Pull request 311](https://github.com/osrf/sdformat/pull/311)
+    * [Pull request 363](https://github.com/osrf/sdformat/pull/363)
+
+1. Param\_TEST: test parsing +Inf and -Inf.
+    * [Pull request 277](https://github.com/osrf/sdformat/pull/277)
```

Comment:
this was in 10.0.0

* https://github.com/osrf/sdformat/commits/sdformat10_10.0.0/src/Param_TEST.cc

probably some of the other ones too

Reply:
Ahh thanks. I didn't see it on 10.x.x below and assumed it was new on 10, but now I realize many of them could have been there since before 10.0.0 :confused: